### ISSUE TITLE
e2e related documentation missing from desktop and mobile client manuals

### ISSUE THREAD

**Issue by sunjam** [nextcloud/end_to_end_encryption#200]
Have tried setting up e2e encryption and cannot find any documentation in the [desktop 3 manual](https://github.com/nextcloud/desktop/issues/2433) or mobile client manuals.  I realize the devs here do not maintain those clients, but wanted to mention it here.  Thanks.

**Comment by sunjam** [comment]
Is it possible for more than one device to use the same key?

**Comment by tobiasKaminsky** [comment]
Yes, that is the whole purpose of the mnemonic to easily set-up e2e devices with the same keys.

**Comment by WuShufei** [comment]
How and where does one file a mnemonic key for a previously set up e2e client?  Specifically, in the Linux desktop client?  I cannot find any field for this in settings.

**Comment by sunjam** [comment]
It is synced through the server via the desktop client iirc

**Comment by joshtrichards** [comment]
It's now in the Desktop manual. There are no manuals for the mobile clients these days. 

But I'll move this over to the main NC documentation repository for follow-up/tracking.
